.vscode/launch.json

```diff
@@ -0,0 +1,54 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug dry run",
+      "type": "Ruby",
+      "request": "launch",
+      "program": "${workspaceRoot}/bin/dry-run.rb",
+      "cwd": "${workspaceRoot}",
+      "useBundler": true,
+      "env": {
+        "LOCAL_GITHUB_ACCESS_TOKEN": ""
```

Comment:
Could we also set this to `${env:LOCAL_GITHUB_ACCESS_TOKEN}` to get the current env value? Similarly, would be useful to also support `LOCAL_CONFIG_VARIABLES` the same way which is currently just a JSON array so not the most user friendly value.

Reply:
I'm adding a commit with the changes required to do that. As long as VS Code is started from an environment which contains one of those environment variables, they will be passed thru to the Docker container. I also made a small improvement to the development Dockerfile so that the `/.vscode-server` folder is mapped to a volume, which will prevent plugins needing to be reinstalled on container rebuild.